pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -597,8 +606,7 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo, provisionC
 			InitContainers:     initContainers,
 			Containers: []v1.Container{
 				{
-					Command:         command,
-					Args:            args,
+					Command:         osdStartScript(command, args),
```

Comment:
is it only applicable for non-pvc osds?

Reply:
Its applicable for both osds on disks as well as pvcs

Reply:
Ohh I see it's common podTemplateSpec